examples/sync/src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub fn create_adb_config() -> Config<Translator> {
         translator: Translator::default(),
         thread_pool: None,
         buffer_pool: commonware_runtime::buffer::PoolRef::new(1024, 10),
+        pruning_gap: 10,
```

Comment:
Seems like an OK value to use during tests? It's sufficiently small such that the "gap" will exist even if we only insert relatively few operations into the database.